docs/how-to/deployment.md

```diff
@@ -11,7 +11,7 @@ ENABLE_AUDIT_LOGS:
 
 The `PUSH_AUDIT_LOGS_CONFIG` field must also be configured.
 
-## Deploying audit-service 1.0.0
+## Deploying audit-service 1.1.0
```

Comment:
no need to change this, the doc describes how to go from `v0.*` to `v1.*`, not specifically to 1.1.0. We can leave it as is or change it to `1.*`